### PR TITLE
Added mutations for sum-oddneven-index and changed the parameter name…

### PR DESCRIPTION
**Pull request by AlerGames**
Added 3 mutations for sum-oddneven-index and changed the parameter name for the function from a to input_list in description.md and starter.py for better readability

### CODE REVIEW

File: sum-oddneven-index/mutation_2.py
Comment (Felix-Egan) [inline sum-oddneven-index/mutation_2.py:1]: Docstring is slightly difficult to read...

File: sum-oddneven-index/mutation_3.py
Comment (Felix-Egan) [inline sum-oddneven-index/mutation_3.py:1]: A good idea actually, didn't think of it.

File: sum-oddneven-index/description.md
Comment (Felix-Egan) [inline sum-oddneven-index/description.md:1]: This improved readability tenfold.

**Review by Felix-Egan — APPROVED** [review]
All and all, looks good. Would add a tiny clarification for mutant 2, though it is still decipherable, but otherwise, code won't crash and will yield unexpected results.

File: sum-oddneven-index/description.md
Comment (AlerGames) [reply sum-oddneven-index/description.md:1]: Thanks!

File: sum-oddneven-index/mutation_3.py
Comment (AlerGames) [reply sum-oddneven-index/mutation_3.py:1]: I was grasping at straws ngl

File: sum-oddneven-index/mutation_2.py
Comment (AlerGames) [reply sum-oddneven-index/mutation_2.py:1]: Enough to warrant a change? Im lazy :pensive:

**Comment by emaj9** [comment]
lgtm

**Pull request merged**